src/Graby.php

```diff
@@ -229,6 +237,8 @@ public function cleanupHtml($contentBlock, $url)
             $html = preg_replace('!</?a[^>]*>!', '', $html);
         }
 
+        $this->logger->debug('Body after cleanupHtml, before cleanupXss:', ['html' => $html]);
```

Comment:
```suggestion
        $this->logger->debug('Body after cleanupHtml, before cleanupXss', ['html' => $html]);
```

Reply:
Resolved in https://github.com/j0k3r/graby/pull/184/commits/0a7d2ca09e7a0f81d9d66ac24b4acad48990d09e